python_modules/dagster/dagster/_core/definitions/asset_graph_subset.py

```diff
@@ -278,8 +278,11 @@ def can_deserialize(
 
         for key, value in serialized_dict["partitions_subsets_by_asset_key"].items():
             asset_key = AssetKey.from_user_string(key)
-            partitions_def = asset_graph.get(asset_key).partitions_def
+            if not asset_graph.has(asset_key):
+                # Asset had a partitions definition at storage time, but no longer does
```

Comment:
minor nit: this should be "Asset had a definition at storage time..."

Reply:
good point, updated